aptos-move/framework/aptos-stdlib/sources/cryptography/ed25519.move

```diff
@@ -35,6 +35,10 @@ module aptos_std::ed25519 {
     // Structs
     //
 
+    struct SecretKey has drop {
```

Comment:
Mark this as `#[test_only]` too, no?

And add comments explaining it's an Ed25519 secret key used to generate signatures during testing.